types.d.ts

```diff
@@ -43,6 +43,12 @@ interface ComponentizeOptions {
    * (no experimental subsystems currently supported)
    */
   enableFeatures?: [],
+  /**
+   * Pass environment variables to the spawned Wizer or Weval Process If set to
+   * true, all host environment variables are passed. To pass only a subset,
+   * provide an object with the desired variables.   
+   */
+  passthroughHostEnv: boolean | Record<string, string>,
```

Comment:
`passthrough` usually means you're passing something directly through from one thing to another, which isn't the case in case of the second type in the union here. `passThrough` as two words might be clearer on that. But maybe let's just call this `hostEnv`?

Also wouldn't `guestEnv` be more correct than `hostEnv`? Or even just `env`?

Reply:
I will go with just `env`. That sounds cleaner.

Reply:
updated.